src/styles/main.scss

```diff
@@ -1,3 +1,9 @@
-body {
-  margin: 0;
-}
+@import "./util/normalize";
+@import "./util/variables";
+@import "./util/function";
```

Comment:
The import path should be './utils/functions', not './util/function'. Make sure the folder and file names are correct

Reply:
isn't fixed